.circleci/config.yml

```diff
@@ -4,24 +4,37 @@ jobs:
 
   build:
     docker:
-      - image: circleci/golang:1.12.1
-    working_directory: /go/src/github.com/roboll/helmfile
+      - image: circleci/golang:1.12.4
+    environment:
+      GOPATH: /go
```

Comment:
@mumoshu you might want to add GO111MODULE=on here. (https://circleci.com/blog/go-v1.11-modules-and-circleci/)

With go modules, we are not bound to GOPATH anymore (yay!).

Reply:
Thanks for the info!

I was thinking that we should rely on the `GO111MODULE=auto` which is the default, so I tweaked `working_directory` rather than `GO111MODULE` like this.

Also `GOPATH=/go` is the default value AFAIK and here to tell readers that the GOPATH is there, so that they can see why `save_cache` is pointing to the paths under `/go`.

Please correct me or add anything if you feel like to do so. Thanks a lot for your review ☺️